### DISCUSSION TITLE
제페토 월드 내, 다른 월드로 바로 이동하는 기능 구현

### DISCUSSION THREAD

**Discussion by FeynLuda** [naverz/zepeto-studio-kor#834] [5. Q&A]
https://world.zepeto.me/ko/detail/qSBuX3wK87QsVBV3MCvvGL0?referrer=copylink_share

위 링크의 월드에는,

다른 월드 (https://world.zepeto.me/ko/detail/rrhmP3wK87qsWmLcOtLtnR5?referrer=copylink_share )로 바로 월드 이동을 하는 기능이 구현되어 있습니다.

같은 기능을 제작 중인 월드에 구현하고자 하는데, 
관련한 가이드를 부탁드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 예시로 들어주신 월드는 빌드잇으로 제작된 제휴 맵이어서 저희가 지원해드린 기능입니다.
유니티로 개발하는 월드에서는 다른 월드로의 이동 기능을 제공하고 있지 않습니다.
양해 부탁 드립니다.
감사합니다.

**Reply by FeynLuda** [reply]
답변 고맙습니다 ^^

지금 제작하는 월드는 build it으로 구현하고 있고, 
빌드잇에 해당 기능이 없어서, 유니티로 할 수 있을지 연구 하던 중이었는데요,

빌드 잇으로 구현한 맵에 해당 기능을 지원해 주실 수 있을 지요?

**Reply by Zepeto-TechEvangelist** [reply]
해당 기능 지원을 위해서는 제휴를 맺어야 할 것 같습니다.

제휴 관련해서 정식으로 메일을 보내보시면 좋을 것 같습니다.
dl_partnerships@naverz-corp.com

검토 및 답변에는 수일이 소요되는 점 양해 부탁 드립니다.
감사합니다.

**Reply by FeynLuda** [reply]
답변 고맙습니다.
검토허여 연락드릴 수 있도록 하겠습니다.
